common/src/crs.rs

```diff
@@ -5,15 +5,17 @@ const G1_START: usize = 28;
 const G2_START: usize = 28 + (5_040_001 * 64);
 const G2_END: usize = G2_START + 128 - 1;
 
+const BACKEND_IDENTIFIER: &str = "acvm-backend-barretenberg";
 const TRANSCRIPT_NAME: &str = "monomial-transcript00.dat";
```

Comment:
```suggestion
const TRANSCRIPT_NAME: &str = "transcript00.dat";
```
Since we have a new directory for UP, let's revert my renaming here.